src/Discord/Internal/Types/Prelude.hs

```diff
@@ -301,15 +285,15 @@ objectFromMaybes = object . catMaybes
 --
 -- Public creation of this datatype should be done using the relevant smart
 -- constructors for Emoji, Sticker, or Avatar.
-data Base64Image a = Base64Image T.Text T.Text
+data Base64Image a = Base64Image { mimeType :: T.Text, base64Data :: B.StrictByteString }
```

Comment:
B.StrictByteString doesn't exist in cabal 8.10 (generating a github actions build error). It's just a type alias for B.ByteString (TIL). 

Should probably use B.ByteString for compatibility?

Reply:
Made this adjustment. Odd that the names are like that, yeah. Personally I'd make StrictByteString the actual name so it's easier to tell it and LazyByteString apart, but I'm not a maintainer there...